bin/start_recurring_jobs.rb

```diff
@@ -1,16 +1,18 @@
-weekday = Time.zone.now.wday
-
-if weekday <= 4
-  days_til_Friday = 5-weekday 	
-else
-	days_til_Friday = 6-weekday+6
+def date_of_next(day)
+  date  = Date.parse(day)
+  delta = date > Date.today ? 0 : 7
+  date + delta
 end
 
-job_time =  Time.now.midnight + (86400*days_til_Friday)
-
+def main
+  job_time = date_of_next("Thursday") + 12.hours
+  jobs = Delayed::Job.where(queue: "clear_groups")
 
-jobs = Delayed::Job.where(queue: "clear_groups")
+  if jobs.length == 0
+    ClearGroupsJob.delay(queue: "clear_groups", run_at: job_time).perform_later
+  end
+end
 
-if jobs.length == 0
-	ClearGroupsJob.delay(queue: "clear_groups", run_at: job_time).perform_later
+if __FILE__ == $0
+  main
```

Comment:
You don't end up seeing this that much in Ruby code.I would just add a `#!/usr/bin/env ruby` at the top, and unwrap `main` and put its code at a high level.